.github/workflows/test.yaml

```diff
@@ -581,7 +635,6 @@ jobs:
       fail-fast: false
       matrix:
         node:
-          - 16.14 # Prisma's minimum is 16.13 but pnpm 8 minimum is 16.14 
```

Comment:
Let's keep this line around but commented so we know how we _should_ be testing this. Otherwise this will get lost sooner or later and we will not explicitly be tested the minimum version any more - and that is important.